docs/guides/high-availability/deploy-infinispan-kubernetes-crossdc.adoc

```diff
@@ -195,24 +195,41 @@ include::examples/generated/ispn-site-b.yaml[tag=infinispan-crossdc]
 +
 {project_name} requires the following caches to be present: `sessions`, `actionTokens`, `authenticationSessions`, `offlineSessions`, `clientSessions`, `offlineClientSessions`, `loginFailures`, and `work`.
 +
+Due to the use of persistent-user-sessions, the configuration of the various `*sessions` caches is different to the `actionTokens`,
```

Comment:
Can you create an issue to track this work. We're currently running our benchmarks with these caches configured with a `maxCount` of 10000.

Reply:
I was not planning to add another boolean to prevent the caches from being configured. We have enough options in taskfiles. They are not used when persistent sessions are enabled, and they are only used with volatile sessions.

Reply:
+1 for creating a follow-up issue for KCB + KC as it would make things simpler for users. Note that the `authenticationSessions` cache is not covered by the persistent user sessions. I've updated the docs.